_layouts/project.html

```diff
@@ -3,7 +3,7 @@
 ---
 
 
-<aside class="usa-width-one-fourth usa-layout-docs-sidenav">
+<aside class="usa-layout-docs-sidenav sidenav">
```

Comment:
Is the class `sidenav` defined somewhere?

Reply:
https://github.com/18F/uswds-jekyll/blob/master/_sass/components/_side-nav.scss#L35